### PR TITLE
Feature/Password/CLK32 957-ChangePassword-Implement-API

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- ChangePassword

### Solution
- update integrate ChangePassword

### Impact
- ChangePassword

### CODE REVIEW

**Comment by namnh92** [comment]
- [ ] I understand that "ChangePassword.ConfirmPassword" is now existed in Localized service

- [ ] I understand that "ChangePassword.Save" is now existed in Localized service

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift [hunk 1]
```diff
@@ -24,27 +24,18 @@ struct ChangePasswordContentView: View {
...
-		 newStyle: TextInputStyle = .default,
-		 confirmStyle: TextInputStyle = .default) {
-		self._currentPassword = .init(initialValue: currentPassword)
-		self._newPassword = .init(initialValue: newPassword)
-		self._confirmPassword = .init(initialValue: confirmPassword)
-		self._currentStyle = .init(initialValue: currentStyle)
-		self._newStyle = .init(initialValue: newStyle)
-		self._confirmStyle = .init(initialValue: confirmStyle)
-	}
+
```
Comment (namnh92) [inline ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/b36012346513a751b0300c2fef0df2c981ef822a/ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift#L39"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>ChangePasswordContentView.swift:39</code></td>
    </tr>
  </tbody>
</table>

**Comment by namnh92** [comment]
- [x] I understand that "ChangePassword.Save" is now existed in Localized service

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift]: fix complete

**Review by namnh92 — APPROVED** [review]

**Pull request merged**